source/includes/mongodb-compatibility-table-java.rst

```diff
@@ -46,7 +56,7 @@
      - |checkmark|
 
    * - 3.12
-     -
+     - |checkmark|  [#3.12-driver-support]_
```

Comment:
We should not indicate compatibility if it doesn't support all the new features. For context, I think we made this agreement in a past Drivers Engineering Leads meeting. This led to the creating of this page: https://docs.mongodb.com/drivers/about-compatibility/

This section describes when checkmarks are appropriate:
> The check marks (✓) indicate that the driver can access all the features of that specific version of MongoDB server unless those features have been deprecated or removed.